src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -247,8 +247,13 @@ public void waitForPanelsToCollapse() {
     public void waitForPanelsToExpand() {
         By panelCollapseSelector = By.cssSelector(".panel-heading+.panel-collapse");
         List<WebElement> webElements = browser.driver.findElements(panelCollapseSelector);
-        
-        waitForElementsVisibility(webElements);
+        for (WebElement element : webElements) {
+            try {
+                waitForElementVisibility(element);
+            } catch (org.openqa.selenium.StaleElementReferenceException e) {
```

Comment:
Let's import `StaleElementReferenceException` and use it directly without its fully qualified name